README.md

```diff
@@ -47,146 +35,37 @@ This package implements the slave side of the simple key-value protocol used  fo
-A ```KiVi``` message is defined as a string encapsulated between double curly braces. The key and value are separated by a semicolon (```;```).
-For example: the ```{{timeout;120}}}``` string is a simple key-value message where the key "_timeout_" is associated with the value "_120_". Both ```greentea-client``` and ```Greentea``` understand this format and can detect key-value messages in a data stream. Both _key_ and _value_ are ASCII strings.
+The key-value protocol (also called ```KiVi```) is a simple protocol introduced to the ```Greentea``` test tools. It is used to send simple text messages (events) between the DUT and the host. Each message consists of a _key_ and corresponding _value_ pair. A ```KiVi``` message is defined as a string encapsulated between double curly braces. The key and value are separated by a semicolon (```;```).
+
+For example, the ```{{timeout;120}}}``` string is a simple key-value message where the key "_timeout_" is associated with the value "_120_". Both ```greentea-client``` and ```Greentea``` understand this format and can detect key-value messages in a data stream. Both _key_ and _value_ are ASCII strings.
 This protocol is a master-slave protocol. The host has the role of _master_ and the DUT is the _slave_.
 
 ```greentea-client``` implements the key-value protocol tokenizer and parser.
 
-## Where can I use it?
-It is possible to write test cases that use ```greentea-client``` and at the same time support ```mbed``` features such as [MINAR scheduler](https://github.com/ARMmbed/minar). It is also possible to mix ```greentea-client``` with other test tools we use at ```mbed``` such as [utest](https://github.com/ARMmbed/utest) and [unity](https://github.com/ARMmbed/unity).
-
-You can also find references to ```greentea-client``` in many ```mbed``` packages. For example:
-  * [mbed-drivers](https://github.com/ARMmbed/mbed-drivers) - check its [module.json](https://github.com/ARMmbed/mbed-drivers/blob/master/module.json)
-  * [core-util](https://github.com/ARMmbed/core-util) - check its [module.json](https://github.com/ARMmbed/core-util/blob/master/module.json)
-
-# Test suite model
-## utest support template
-```c++
-#include "greentea-client/test_env.h"
-#include "utest/utest.h"
-#include "unity/unity.h"
-
-void test_case_1_func() {
-    // Test case #1 body
-    // Here you can run your test cases and assertions
-    TEST_ASSERT_TRUE(true);
-    TEST_ASSERT_FALSE(false);
-}
-
-void test_case_2_func() {
-    // Test case #2 body
-    // Here you can run your test cases and assertions
-    TEST_ASSERT_TRUE(true);
-    TEST_ASSERT_FALSE(false);
-}
-
-const Case cases[] = {
-    Case("Test case #1 name", test_case_1_func),
-    Case("Test case #1 name", test_case_2_func)
-};
+# Test suite model with greentea-client
```

Comment:
We should link to the example code instead of have untested code provided in the README.